app/controllers/ops_controller/settings.rb

```diff
@@ -153,6 +153,7 @@ def region_edit
       @edit[:region].description = @edit[:new][:description]
       begin
         @edit[:region].save!
+        MiqRegion.my_region_clear_cache
```

Comment:
Agreed. I don't like it here either. Would it be better to use an `after_save` callback to clear the cache? I'm not sure what the consensus is on callbacks ...